README.md

```diff
@@ -4,9 +4,9 @@
 [![Build Status](https://github.com/ViaVersion/ViaBackwards/actions/workflows/gradle.yml/badge.svg?branch=master)](https://github.com/ViaVersion/ViaBackwards/actions)
 [![Discord](https://img.shields.io/badge/chat-on%20discord-blue.svg)](https://viaversion.com/discord)
 
-**Allows the connection of older clients to newer server versions for Minecraft servers.**
+**Allows older clients to connect to newer server versions for Minecraft servers.**
 
-Requires [ViaVersion](https://hangar.papermc.io/ViaVersion/ViaVersion) to be installed..
```

Comment:
I think this was intended as many people forget to install ViaVersion

Reply:
The second period being removed is an OK change.